components/TARGET_PSA/TARGET_TFM/COMPONENT_SPE/secure_fw/spm/spm_db.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2017-2019, Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+
+#ifndef __SPM_DB_H__
+#define __SPM_DB_H__
+
+#ifdef TFM_PSA_API
+#include "tfm_thread.h"
+#endif
+
+struct spm_partition_desc_t;
+struct spm_partition_db_t;
+
+uint32_t get_partition_idx(uint32_t partition_id);
+
+typedef int32_t(*sp_init_function)(void);
+
+#define TFM_PARTITION_TYPE_APP   "APPLICATION-ROT"
+#define TFM_PARTITION_TYPE_PSA   "PSA-ROT"
+
+#define TFM_STACK_SIZE  1024
```

Comment:
Should TFM_STACK_SIZE be configurable, driven from config files? On what basis we are hardcoding this to 1024?

Reply:
Actually each partitions' stack size is configured in their manifest, this is one of our patches to TFM code, see commit fc7864055982f3b8e7e556f9dd8d0c94a1c772bc, so `TFM_STACK_SIZE` is not used anymore.